fusor-ember-cli/app/controllers/review/summary.js

```diff
@@ -19,6 +19,11 @@ export default Ember.Controller.extend({
     return ('http://' + this.get('model.openstack_undercloud_ip_addr'));
   }.property('model.openstack_undercloud_ip_addr'),
 
+  overcloudUrl: function() {
+    return ('http://' + this.get('model.openstack_overcloud_password') + '/dashboard/admin');
```

Comment:
Are you saving the full URL including /dashboard/admin in the DB. I assume that you were not so I added here. Is this correct? @jmontleon